tests/unit/services/sl-modal-test.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember';
+import { moduleFor, test } from 'ember-qunit';
+import ModalService from 'sl-ember-components/services/sl-modal';
+
+let service = ModalService.create();
+
+let MockModal = Ember.Object.extend({
+    name: 'mock',
+    isOpen: true
+});
+
+moduleFor( 'service:sl-modal', 'Unit | Service | sl modal', {
+    unit: true,
+
+    beforeEach() {
+        //reset modals property before each test so tests are independent
+        service.set( 'modals', {} );
+    }
+});
+
+test( 'Register a modal and find it', function( assert ) {
```

Comment:
This is not isolated enough.  If this test fails we do not immediately know why - is it because the registration logic is failing or the find logic?